packages/rich-text/package.json

```diff
@@ -22,11 +22,21 @@
 	"react-native": "src/index",
 	"dependencies": {
 		"@babel/runtime": "^7.4.4",
+		"@wordpress/blob": "file:../blob",
```

Comment:
These changes make me think we need to update Core to support the webpack plugin to extract the dependencies, otherwise, we could miss these changes once we update Core. cc @gziolo @sirreal

Reply:
Sorry for overlooking this. Is there an open issue somewhere to track this or can you provide more details? I don't know what would be involved in that work.

Reply:
What's different here from other dependencies being added to packages?